Sample_TCs.txt

```diff
@@ -0,0 +1,154 @@
+sample_document = {
```

Comment:
Please make it a json file instead of text file, and read from that, in case this is that file which is the input file for the project

Reply:
Premal
This is for ritiks reference , in next pr I will remove it add json file by EOD today

Reply:
cool